doc/release/1.16.0-notes.rst

```diff
@@ -72,6 +72,8 @@ comparisons.
 C API changes
 =============
 
+The ``NPY_API_VERSION`` was incremented to 0x0000D since the `PyUFuncObject`'s
+size changed.
```

Comment:
This is not the reason - it incremented because `PyUFuncObject.core_dim_flags` and `PyUFuncObject.core_dim_sizes` were added to the API.

Reply:
fixed